### PR TITLE
refactoring/linting/vetting/idiomatic

### PR DESCRIPTION
**Pull request by hydroflame**
Mostly learning to use your lib, feel free to reject if it's not to your standard

### CODE REVIEW

**Comment by hydroflame** [comment]
also there didn't seem to be any tests so I ran al the programs and they all worked

File: chaos/chaos.go [hunk 1]
```diff
@@ -8,20 +8,28 @@ package main
...
 
 func main() {
 	if len(os.Args) != 2 {
 		fmt.Printf("%s NAMESERVER\n", os.Args[0])
 		os.Exit(1)
 	}
-	conf, _ := dns.ClientConfigFromFile("/etc/resolv.conf")
+	conf, err := dns.ClientConfigFromFile("/etc/resolv.conf")
+	if err != nil {
+		fmt.Println("error making client from default file", err)
```
Comment (miekg) [inline chaos/chaos.go]: potentially a log.Fatalf here as well?

File: check-soa/check-soa.go [hunk 2]
```diff
@@ -5,14 +5,15 @@ package main
...
 	"os"
 	"strings"
 	"time"
+
+	"github.com/miekg/dns"
 )
 
 const (
-	TIMEOUT time.Duration = 5 // seconds
+	defaulttimeout time.Duration = 5 * time.Second
```
Comment (miekg) [inline check-soa/check-soa.go]: Uppercase the second defaultTimeout, or just Timeout

File: q/q.go [hunk 3]
```diff
@@ -516,9 +515,9 @@ func denial3(nsec3 []dns.RR, in *dns.Msg) {
 		return
 	case dns.RcodeNameError: // NXDOMAIN Proof
 		indx := dns.Split(qname)
-		ce := "" // Closest Encloser
-		nc := "" // Next Closer
-		wc := "" // Source of Synthesis (wildcard)
+		var ce string // Closest Encloser
```
Comment (miekg) [inline q/q.go]: why this change?

**Comment by miekg** [comment]
Looks nice. Thanks! some tiny nits and then we can merge.

Yes, there should be (hermetic) tests...

File: q/q.go (hunk 3 above)
Comment (NewRegin) [inline q/q.go]: the "ce := """ style  is better in golang, see Golang lib@hydroflame

Comment (miekg) [inline q/q.go]: But you changed it from `:=` to `var`? #confused

Comment (hydroflame) [inline q/q.go]: $ grep -r "var .* string" . | wc -l
     450
$ grep -r ".* := \"\"" . | wc -l
     181

File: chaos/chaos.go (hunk 1 above)
Comment (hydroflame) [inline chaos/chaos.go]: Your call really, but the rest of the program is using fmt.Prin* and then os.Exit
